Modules/_lzmamodule.c

```diff
@@ -24,6 +24,14 @@
     #error "The maximum block size accepted by liblzma is SIZE_MAX."
 #endif
 
+
+#ifndef LZMA_FILTER_ARM64
```

Comment:
I have no idea why the new constants are defined here while the existing ones are not.

Reply:
This is for compatibility purpose.
I'm trying to define the constants here to avoid potential build failure.
BCJ filters other than ARM64 and RISCV had been there since the Python lzma module started to exist while the added new filters require lzma v5.4 and v5.6 respectively. If not doing so, building on targets of which lzma version is old will fail.
Also doing so is safe as the lzma library maintains ABI stability: If Python is built with old or new lzma C headers and paired with new runtime lzma dynamic library, everything will just work fine; If Python is built with old or new lzma C headers and paired with old dynamic library, using the new ARM64 or RISCV filter will raise runtime errors while other usage will still work.

Reply:
Thanks. I added a clarifying comment.